deepchem/data/data_loader.py

```diff
@@ -415,29 +384,119 @@ def _get_shards(self, input_files, shard_size):
       List of filenames to process
     shard_size: int
       The size of a shard of data to process at a time.
+
+    Returns
+    -------
+    Iterator over shards
```

Comment:
In the case of Numpy docstring style, the first line is type annotation.
https://numpydoc.readthedocs.io/en/latest/format.html



```
Iterator[pd.DataFrame]
  Iterator over shards
```

Reply:
Fixed!